templates/hardwarelab/event_manager.html

```diff
@@ -0,0 +1,192 @@
+{% extends "layout.html" %}
+{% block content %}
+<script>
+$(document).ready(function(){
+    $(".create-rental").on("click", function() {
+      $rental_id = $(this).siblings("#rental-id").attr("value");
+      $inventory_id = $(this).parentsUntil('#reservations').find('#inventory_id').val();
+      console.log($inventory_id);
+      if($inventory_id == ''){
+        console.log('empty');
+        alertify.alert("Please select an Inventory ID");
+      } else {
+        $("#create-rental-modal").modal({
+          closable  : true,
+          onDeny    : function(){
+            return true;
+          },
+          onApprove : function() {
+            $.ajax({
+              url : "/api/rent/", // the endpoint
+              type : "GET", // http method
+              data : { 
+                rental_id: $rental_id,
+                inventory_id: $inventory_id
+              },
+              success : function(data) {
+                location.reload();
+                // $("#manager-container").load(document.URL + ' #manager-container')
+              }, 
+              error: function(data) {
+                alertify.alert(data.responseText);
+              }
+            });
+          },
+        }).modal('show');
+      }
+    });
+    $(".return").on("click", function() {
+      $rental_id = $(this).siblings("#rental-id").attr("value");
```

Comment:
this is **very bad**  !!

"#rental-id"  , the id needs to be unique! that's why with jquery, you will always get one element when doing this $('#rental-id') , whereas multiple elements when doing $('.rental-id')

If the id must appear multiple time, it is class not an id!